doc/recognition.xml

```diff
@@ -336,8 +336,11 @@ set this to <K>false</K>. The result is either <K>fail</K> or
 <K>false</K> if the method fails or a recognition info record <C>ri</C>.
 If the method created a leaf then <C>ri</C> will be a leaf, otherwise
 it will have the attribute <Ref Attr="Homom"/> set, but no factor or
-kernel have been created or recognised yet. You can use for example
-the methods in <Ref Var="FindHomMethodsPerm"/> or
+kernel have been created or recognised yet. Optionally,
+<Ref Attr="HomomSourceCheck"/> can be set. When this is set, <Ref Attr="Homom"/>
```

Comment:
Hmm, I am not in love with that name. It sounds to me as if the source was checked. But rather, it is some input that is meant to be passed to the homomorphism that is checked, or validated.
So perhaps `ValidateHomomInput` or `CheckHomomInput`? Or `IsValidInputForHomom`?

Reply:
I like IsValidInputForHomom

Reply:
although actually, normally IsX properties in GAP return true/false, so maybe ValidateHomomInput would be better...

Reply:
IMO a little bit better than `ValidateHomomInput` would be `ValidateInputForHomomorphism`.

Reply:
Regarding the name: as long as I can use a global search&replace to adjust it, I am not going to spend extended amounts of time arguing about it ;-). We of course need to carefully think about all those names, but that can happen in a follow-up PR, no need to delay the fix in this PR for it.

Reply:
I think the introduction of validatehomominput and ValidateHomomInput makes the naming particularly horrible, but that's horrible consistent naming, and I'm very happy for it to be changed so we don't rely on case :)